docs/modules/rust-guide/pages/rust-quickstart.adoc

```diff
@@ -121,43 +82,79 @@ cd rust_hello
 
 == Modify the default configuration
 
-Creating a new project adds a default `+dfx.json+` configuration file to your project directory.
-This file contains settings required to build a project for the {IC} much like the `+Cargo.toml+` file provides build and package management configuration details for Rust programs.
-You need to modify the default settings in both of these files to build Rust programs that run on the {IC} as canisters.
+Creating a new project adds several template files to your project directory much like when you create a new Rust package using the `+cargo new+` command.
+You need to modify these default files and add some Rust-specific files to your project before you can build canisters that run on the {IC}.
 
-=== Delete the default program
+=== Replace the default program
 
-Creating a new project creates a default `+src+` directory with a template `+main.mo+`. We'll eventually replace this with a `+lib.rs+` file. 
+Creating a new project creates a default `+src+` directory with a template `+main.mo+`. We'll replace this default program with a Rust library `+lib.rs+` package and `+Cargo.toml+` files. 
 
-. Delete the `+main.mo+` file. 
+To replace the default program:
 
-. In the terminal, run the following command:
+. In the terminal, check that you are in the root directory for your project, if needed.
+. Delete the template `+main.mo+` file by running the following command:
++
+[source,bash]
+---- 
+rm src/rust_hello/main.mo
+----
+. Create a new cargo package using a library template by running the following command:
 +
 [source, bash]
 ----
 cargo init --lib
 ----
++
+This command creates a `+src+` directory with a library (`+lib.rs+`) package and a `+Cargo.toml+` file in the project directory.
 
-=== Edit cargo settings
+=== Edit Cargo settings for the project
 
-Modify the `+Cargo.toml+` configuration file:
+To modify `+Cargo.toml+` configuration file for the project:
 
 . Check that you are still in the source directory for your project, if needed.
-. Open the default `+Cargo.toml+` file.
-. Configure settings for your project.
+. Open the default `+Cargo.toml+` file you created by running the `+cargo init+` command in a text editor.
```

Comment:
If you follow my last comments, then I think it makes sense here to *create* a new Cargo.toml at the project root (`./Cargo.toml`).